### PR TITLE
added the rest of setup options to sync and changed map display on cl…

### PR DESCRIPTION
**Pull request by PieroganoReggiano**
…ient side

* added more synchronization -- now all used options in world setup are synchronized
* also, i added some client side tweaks to world setup -- now map is only displayed and cannot be changed there
* game_setup_info world map name is now only file name, not full path -- there were some places other where it was done this way, so it was easier to make this everywhere

### CODE REVIEW

**Review by NamespaceV — APPROVED** [review]

File: Scripts/UI/multi_game_setup.gd
Comment (NamespaceV) [inline Scripts/UI/multi_game_setup.gd:1]: `client_side` field has source of truth issues vs `IM.get_client()`

ok for now but code smell to clean up

**Review by NamespaceV — APPROVED** [review]

**Comment by NamespaceV** [comment]
success_meme

File: Scripts/Multiplayer/all_the_commands.gd
[the remark this answers is not in the document]
Comment (PieroganoReggiano) [reply Scripts/Multiplayer/all_the_commands.gd:1]: of course

File: Scripts/UI/multi_game_setup.gd
Comment (PieroganoReggiano) [reply Scripts/UI/multi_game_setup.gd:1]: Kind of, but `client_side` does not depend on `IM.get_client()`. It depends on the UI it is created by -- `HostLobby` or `ClientLobby` and this for now is not 1:1 with `IM.get_client()`

**Pull request merged**